<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaLicencia</class>
 <widget class="QDialog" name="VentanaLicencia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activación de Licencia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/iconos/icono.ico</normaloff>../assets/iconos/icono.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelSerial">
     <property name="text">
      <string>Serial:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineSerial"/>
   </item>
   <item>
    <widget class="QLabel" name="labelClave">
     <property name="text">
      <string>Clave:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineClave"/>
   </item>
   <item>
    <widget class="QCheckBox" name="checkFechaManual">
     <property name="text">
      <string>Usar fecha de expiración manual</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dateExpiracion">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDias">
     <item>
      <widget class="QLabel" name="labelDias">
       <property name="text">
        <string>Días de validez:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinDiasValidez">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="suffix">
        <string> días</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>3650</number>
       </property>
       <property name="value">
        <number>365</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBotones">
     <item>
      <widget class="QPushButton" name="btnActivar">
       <property name="text">
        <string>Activar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExtender">
       <property name="text">
        <string>Extender</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
